entries/d/dotloop.com.json

```
@@ -0,0 +1,14 @@
+{
+  "Dotloop": {
+    "domain": "dotloop.com",
+    "tfa": [
+      "call",
+      "sms",
+      "totp"
+    ],
+    "documentation": "https://support.dotloop.com/s/article/Two-Factor-Authentication",
+    "categories": [
+      "task"
```

Dotloop doesn't seem to be a task/to-do list service?

```suggestion
      "other"
```

---

Thank you very much for your tips. I have modified the SVG and categories. Please review it again. Thank you very much again!